### DISCUSSION TITLE
Peek into file for the workspace editor

### DISCUSSION THREAD

**Discussion by SenorRagequit** [exorstudios/riftbreaker-tools#136] [Wishlist] [answered]
**Wish**
---
Being able to peek inside a file in the workspace editor

**Why**
---
Normally you look into a file to copy some lines for the actual mod, or you need to find a path inside that file or whatever.
Currently, you have to copy that file into the workspace, open it, look/copy stuff, delete it from the workspace again.
Possible solution:
Create a "peek into file" option for *.ent and *.lua files where it gets opened in some read only mode and the file itself gets placed into the temp files of the operating system

**Comment by lukaasm** [comment, marked as answer]
Promoted to feature request
